tests/tensor/rewriting/test_linalg.py

```diff
@@ -545,3 +546,15 @@ def test_svd_uv_merge():
             assert node.op.compute_uv
             svd_counter += 1
     assert svd_counter == 1
+
+
+def test_inv_inv_rewrite():
```

Comment:
Parameterize this with `inverse_op`, and add one case for each Op you support (`MatrixInverse`, `MatrixPInv` so far)